### PR TITLE
contact form markup and styling

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: theme/sass/components/_contact-form.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  h2 {
+    text-align: center;
+    margin: 0;
+  }
+
+  input[type=text], select, textarea {
+    width: 100%;
+    padding: 12px;
+    border: 1px solid $secondary-color;
```
Comment (ElXyfer) [inline theme/sass/components/_contact-form.scss]: Make border colour a lighter grey.

File: theme/sass/components/_contact-form.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  padding: 20px;
+  width: $gutter-size;
+  margin: 0 auto;
+
+  h2 {
+    text-align: center;
+    margin: 0;
+  }
+
+  input[type=text], select, textarea {
```
Comment (ElXyfer) [inline theme/sass/components/_contact-form.scss]: Move everything from line 13 into global sass file.

File: theme/sass/components/_contact-form.scss (hunk 1 above)
Comment (katlazarova) [reply theme/sass/components/_contact-form.scss]: done

File: theme/sass/components/_contact-form.scss (hunk 2 above)
Comment (katlazarova) [reply theme/sass/components/_contact-form.scss]: done

**Pull request merged**